src/App.tsx

```diff
@@ -1,51 +1,13 @@
-import  { useState, useEffect } from 'react';
+import { Todos } from './components/Todo';
 import './index.css';
 
 function App() {
-  const [x, setX] = useState([]); // Malos nombres de variables
```

Comment:
Inconsistent and Non-Descriptive Variable Names: Variable names such as x and y are not descriptive and do not convey their purpose.  For this case you could use names as todos, and setTodos.